data/learnsets.js

```diff
@@ -68873,6 +68873,50 @@ let BattleLearnsets = {
 		venomdrench: ["8M"],
 		venoshock: ["8M", "8L16"],
 	}},
+	eternatuseternamax: {learnset: {
```

Comment:
Does it actually have learnset differences

Reply:
No

Reply:
Then there's no need for a learnset. Especially since this is only usable in Custom Game and everything is legal.